### PR TITLE
Some homepage styling issue fixes

### LINKED ISSUE #87: Address homepage styling issues
Reference page: https://personal.nedbank.co.za/home.html
Franklin page: https://main--nedbank--hlxsites.hlx.page/content/nedbank/za/en/personal/home

List of issues to be addressed

- Navbar background share cropped 
image

- Incorrect banner shade 
image

- Incorrect text font and style
image

- Align footer style
image
image

### PR DESCRIPTION
**Pull request by bosschaert**
Some fixes for the homepage styling, addresses:
* banner shade
* switching section styling
* promotion header and arrow location
* added `-webkit-font-smoothing: antialiased;` to the root style to make the font look better (this setting is also in the original website)

It does not yet address the navbar popup styling issues.

Fix #87

Test URLs:
- Before: https://main--nedbank--hlxsites.hlx.page/content/nedbank/za/en/personal/home
- After: https://issue-87--nedbank--hlxsites.hlx.page/content/nedbank/za/en/personal/home

### CODE REVIEW

**Comment by bosschaert** [comment]
I created this as a draft PR because I'm not 100% sure about some things yet.
Because the font is slightly different from the original things cannot be made to look 100% identical.

Also some text pieces have no style class associated with it, therefore I used just the`#id` syntax to refer to them. Is that considered ok? 

Any other suggestions?

File: blocks/cards/cards.less [hunk 1]
```diff
@@ -855,6 +855,37 @@ autocomment();
...
 }
 
+#switch-to-nedbank-hassle-free {
+  font-weight: 400;
+  line-height: 32px;
+  font-size: 24px;
+
+  + p {
+    font-size: 16px;
+    line-height: 24px;
```
Comment (vdua) [inline blocks/cards/cards.less:865]: would suggest to use less variables. As an example for font-size https://github.com/hlxsites/nedbank/blob/main/styles/variables.less#L84

**Review by vdua — CHANGES_REQUESTED** [review]

File: blocks/cards/cards.css [hunk 2]
```diff
@@ -596,6 +596,31 @@
     display: flex;
   }
 }
+#switch-to-nedbank-hassle-free {
```
Comment (sdmcraft) [inline blocks/cards/cards.css:599]: Using this kind of selector could be problematic. If the author changes the text in the word doc, this selector would no longer work. Maybe use another selector to  achieve the same. Also I see that this is a `h3` . Does it need to be styled differently from the  global `h3` styles?

File: blocks/cards/cards.less (hunk 1 above)
Comment (bosschaert) [reply blocks/cards/cards.less:865]: Good point, thanks @vdua ! I will update the PR with that.

**Comment by bosschaert** [comment]
Thanks for the feedback every @vdua and @sdmcraft - I think I addressed your concerns with these changes now.

The dropdown menu styling issue is not yet addressed. I'll look at that separately.

**Review by sdmcraft — APPROVED** [review]

**Pull request merged**